src/main/java/AN_hw_4/Task1.java

```diff
@@ -0,0 +1,40 @@
+package AN_hw_4;
+
+import java.util.Scanner;
+
+public class Task1 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter the coefficients a, b, and c");
+        System.out.println("a: ");
+        double a = scanner.nextDouble();
+        System.out.print("b: ");
+        double b = scanner.nextDouble();
+        System.out.print("c: ");
+        double c = scanner.nextDouble();
+        double D = b * b - 4 * a * c;
```

Comment:
bad name for 'D' variable. First of all - all variables should begin from the small letter.
Lets rename it as discriminator

Reply:
fixed